CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+Any contribution that you make to this repository will
```

Comment:
Let's talk about it.
It's weird that it starts from 5, lets create a fill contributions file (See LibRealSense)
Github can help with a template

Reply:
Aligned it with librealsense contributing file